keyboards/maartenwut/wonderland/rules.mk

```diff
@@ -28,3 +28,5 @@ UNICODE_ENABLE = no 		# Unicode
 BLUETOOTH_ENABLE = no # Enable Bluetooth with the Adafruit EZ-Key HID
 AUTO_SHIFT_ENABLE = no
 VELOCIKEY_ENABLE = yes
+VIA_ENABLE = yes
+LTO_ENABLE = yes
```

Comment:
The convention is to add a `via` keymap, and not enable for all keymaps
```suggestion
```

Reply:
ah, sorry about that! I will make the changes and make a via keymap folder with rules.mk and the modified default keymap. Thank you!